tests/sles4sap/qesapdeployment/destroy.pm

```diff
@@ -14,10 +14,13 @@ use qesapdeployment;
 sub run {
     select_serial_terminal;
 
-    my $ret = qesap_execute(cmd => 'ansible', cmd_options => '-d', verbose => 1, timeout => 300);
-    die "'qesap.py ansible -d' return: $ret" if ($ret);
-    $ret = qesap_execute(cmd => 'terraform', cmd_options => '-d', verbose => 1, timeout => 1200);
-    die "'qesap.py terraform -d' return: $ret" if ($ret);
+    my $ansible_ret = qesap_execute(cmd => 'ansible', cmd_options => '-d', verbose => 1, timeout => 300);
+    if ($ansible_ret) {
+        qesap_cluster_logs();
+    }
+    my $terraform_ret = qesap_execute(cmd => 'terraform', cmd_options => '-d', verbose => 1, timeout => 1200);
```

Comment:
Oh, I like this! Old way was not calling on `terraform destroy` if the ansible cleanup failed. Good catch!